src/Microsoft.AspNetCore.SignalR.Common/Internal/Protocol/MessagePackHubProtocol.cs

```diff
@@ -144,20 +153,23 @@ private void WriteMessageCore(HubMessage message, Stream output)
             switch (message)
             {
                 case InvocationMessage invocationMessage:
-                    WriteInvocationMessage(invocationMessage, packer, output);
+                    WriteInvocationMessage(invocationMessage, packer);
```

Comment:
:+1: - not sure why there were no warnings.

Reply:
I don't think that analysis is on by default, it was an FxCop rule back in the day.